sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyReleasePolicy.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class that contains the policy rules under which the key can be exported.
+ */
+@Fluent
+public class KeyReleasePolicy {
+    /**
+     * Content type and version of key release policy. The default value is:
+     * 'application/json; charset=utf-8; version=1.0'.
+     */
+    @JsonProperty(value = "contentType")
+    private String contentType = "application/json; charset=utf-8; version=1.0";
```

Comment:
You also might not want to encode this. It's tight coupling. Sending null is fine (not required), in which case this is what the server should default to.